### PR TITLE
Fixes Form Input Clear

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith Please Review

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

File: src/form/Form.js [hunk 1]
```diff
@@ -86,7 +86,9 @@
     Form.prototype.clear = function(){
         var inpArr = this.inputs();
         inpArr.forEach(function(inp){
-            if(inp.type() === "checkbox"){
+            if (inp instanceof Slider) {
+                inp.value(" ");
```
Comment (GordonSmith) [inline src/form/Form.js]: inp.value(inp.low()) would be better?

Comment (mlzummo) [inline src/form/Form.js]: Yes

**Comment by mlzummo** [comment]
Fixed

**Comment by GordonSmith** [comment]
Thx.

**Pull request merged**